src/features/rewards/box/index.tsx

```diff
@@ -74,7 +73,7 @@ export default class Box extends React.PureComponent<Props, State> {
     } = this.props
 
     return (
-      <StyledWrapper id={id}>
+      <StyledCard testId={id}>
```

Comment:
Should this one have an `id` as well?

Reply:
`testId` is required by `Card` component